auth0-lock.js

```diff
@@ -33,6 +33,11 @@ class Auth0Lock {
     }
     LockModule.authenticate(connectionName, options, callback);
   }
+
+  getDelegationToken(options, callback) {
```

Comment:
Same as in `A0LockReactModule`, let's call it just `delegation(options, callback)`